libtiff/libtiff_ctypes.py

```diff
@@ -966,6 +995,58 @@ def WriteEncodedStrip(self, strip, buf, size):
         assert r.value == size, repr((r.value, size))
     writeencodedstrip = WriteEncodedStrip
 
+    @debug
+    def ReadTile(self, buf, x, y, z, sample):
+        """ Read and decode a tile of data from an open TIFF file
+
+        Parameters
+        ----------
+        buf: array
+            contiguous array of bytes
+        x: int
```

Comment:
Indicate that buf is an array which must already be allocated and be sufficiently big to hold an entire tile (cf TIFFTileSize()).
Indicate that x and y are _pixel_ coordinates, and the tile containing that pixel will be returned.